next/.env.development

```diff
@@ -17,3 +17,4 @@ NEXT_PUBLIC_MEILISEARCH_SEARCH_API_KEY=see-instructions-in-readme
 NEXT_PUBLIC_CLOUDFLARE_TURNSTILE_SITE_KEY=0x4AAAAAAAA0ntsHQvjN-Lua
 CLOUDFLARE_TURNSTILE_SECRET_KEY=get-this-from-cf
 NEXT_PUBLIC_CLOUDFLARE_TURNSTILE_SITEVERIFY_API=https://challenges.cloudflare.com/turnstile/v0/siteverify
+REVaLIDATE_SECRET_TOKEN=
```

Comment:
`a` -> `A`
You could use copy&paste for variable names to avoid this issues. :D